docs/schema.md

```diff
@@ -0,0 +1,186 @@
+# Schema Registry
+
+KoP has implemented the Schema Registry to support Kafka clients with [Confluent's Schema serializers and deserializers](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#supported-formats).
+
+To enable the schema registry, you should add the following configuration:
```

Comment:
Just like other KoP configurations. If you start a broker with KoP, add it to `broker.conf`. If you start a standalone with KoP, add it to `standalone.conf`. So I think it would be redundant to mention which `.conf` file should it be added to.